extensions/panache/hibernate-orm-rest-data-panache/deployment/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-links-deployment</artifactId>
+            <scope>test</scope>
```

Comment:
I had to explicitly add this dependency, I guess because we had to use the `-deployment` version. However, this is not necessary to be added when using `quarkus-hibernate-orm-rest-data-panache` (there are tests we verify this).